src/components/SettingsForm/index.jsx

```diff
@@ -63,14 +52,14 @@ export default function SettingsForm({ isWelcome = false }) {
         />
       )}
       <Button
-        className={cls(margin.mt4, {
+        className={cls(margin.mt6, {
           [type.textXl]: isWelcome,
           [padding.px8]: isWelcome,
         })}
         variant="solid"
         onClick={handleClick}
       >
-        {isWelcome ? 'Start' : 'Save'}
```

Comment:
Can we add a Sign Out button which resets the Auth? Should be side by side and only active if isWelcome is not true.